test/libyul/evmCodeTransform/eof_dup256.yul

```diff
@@ -0,0 +1,2594 @@
+        sstore(115, v115)
+        sstore(114, v114)
+        sstore(113, v113)
+        sstore(112, v112)
+        sstore(111, v111)
+        sstore(110, v110)
+        sstore(109, v109)
+        sstore(108, v108)
+        sstore(107, v107)
+        sstore(106, v106)
+        sstore(105, v105)
+        sstore(104, v104)
+        sstore(103, v103)
+        sstore(102, v102)
+        sstore(101, v101)
+        sstore(100, v100)
+        sstore(99, v99)
+        sstore(98, v98)
+        sstore(97, v97)
+        sstore(96, v96)
+        sstore(95, v95)
+        sstore(94, v94)
+        sstore(93, v93)
+        sstore(92, v92)
+        sstore(91, v91)
+        sstore(90, v90)
+        sstore(89, v89)
+        sstore(88, v88)
+        sstore(87, v87)
+        sstore(86, v86)
+        sstore(85, v85)
+        sstore(84, v84)
+        sstore(83, v83)
+        sstore(82, v82)
+        sstore(81, v81)
+        sstore(80, v80)
+        sstore(79, v79)
+        sstore(78, v78)
+        sstore(77, v77)
+        sstore(76, v76)
+        sstore(75, v75)
+        sstore(74, v74)
+        sstore(73, v73)
+        sstore(72, v72)
+        sstore(71, v71)
+        sstore(70, v70)
+        sstore(69, v69)
+        sstore(68, v68)
+        sstore(67, v67)
+        sstore(66, v66)
+        sstore(65, v65)
+        sstore(64, v64)
+        sstore(63, v63)
+        sstore(62, v62)
+        sstore(61, v61)
+        sstore(60, v60)
+        sstore(59, v59)
+        sstore(58, v58)
+        sstore(57, v57)
+        sstore(56, v56)
+        sstore(55, v55)
+        sstore(54, v54)
+        sstore(53, v53)
+        sstore(52, v52)
+        sstore(51, v51)
+        sstore(50, v50)
+        sstore(49, v49)
+        sstore(48, v48)
+        sstore(47, v47)
+        sstore(46, v46)
+        sstore(45, v45)
+        sstore(44, v44)
+        sstore(43, v43)
+        sstore(42, v42)
+        sstore(41, v41)
+        sstore(40, v40)
+        sstore(39, v39)
+        sstore(38, v38)
+        sstore(37, v37)
+        sstore(36, v36)
+        sstore(35, v35)
+        sstore(34, v34)
+        sstore(33, v33)
+        sstore(32, v32)
+        sstore(31, v31)
+        sstore(30, v30)
+        sstore(29, v29)
+        sstore(28, v28)
+        sstore(27, v27)
+        sstore(26, v26)
+        sstore(25, v25)
+        sstore(24, v24)
+        sstore(23, v23)
+        sstore(22, v22)
+        sstore(21, v21)
+        sstore(20, v20)
+        sstore(19, v19)
+        sstore(18, v18)
+        sstore(17, v17)
+        sstore(16, v16)
+        sstore(15, v15)
+        sstore(14, v14)
+        sstore(13, v13)
+        sstore(12, v12)
+        sstore(11, v11)
+        sstore(10, v10)
+        sstore(9, v9)
+        sstore(8, v8)
+        sstore(7, v7)
+        sstore(6, v6)
+        sstore(5, v5)
+        sstore(4, v4)
+        sstore(3, v3)
+        sstore(2, v2)
+        sstore(1, v1)
+        sstore(0, v0)
+    }
+}
+// ====
+// EVMVersion: >=osaka
+// bytecodeFormat: >=EOFv1
```

Comment:
This is redundant. These tests just depend on EOF, regardless of which fork it ultimately gets released in.

```suggestion
// bytecodeFormat: >=EOFv1
```